.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+# Prerequisites
+
+```Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**```
```

Comment:
Why surrounding most of the lines with back ticks? That way, they will appear as code snippets.

Reply:
Oh ok. I just felt that they appear cleaner when seen in MarkDown Preview. So that's why. But sure I'll remove them. Do you want them to be removed from all the places apart from where they represent actual Code Snippets ?